src/libraries/System.Text.Json/src/ILLink/ILLink.Substitutions.xml

```diff
@@ -0,0 +1,11 @@
+<linker>
+  <assembly fullname="System.Text.Json">
+    <type fullname="System.Text.Json.AppContextSwitchHelper">
+      <method signature="System.Boolean get_UseReflectionDefault()" body="stub" value="true" 
+              feature="System.Text.Json.Serialization.UseReflectionDefault" featurevalue="true"/>
+
+      <method signature="System.Boolean get_IsSourceGenReflectionFallbackEnabled()" body="stub" value="true"
+              feature="System.Text.Json.Serialization.EnableSourceGenReflectionFallback" featurevalue="true"/>
```

Comment:
I'm not sure there is value in putting this switch in the substitutions file.

1. You will also need to define the MSBuild `RuntimeHostConfigurationOption` item with `Trim=true` for this, or else it won't get passed to the trimmer.
2. There won't be any code that is trimmed by setting this value to `true`. This appears to be the only usage of it:
https://github.com/dotnet/runtime/blob/967250cc86b4c613015066b47b882e7424fdb590/src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs#L696-L699
3. It would actually make more sense to change the bool property to `false` when the feature switch value isn't set at all, but that runs into https://github.com/dotnet/linker/issues/3176.

So I'd suggest just removing this all together from the Substitutions file.